shared/templates/static/ansible/sshd_set_keepalive.yml

```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_rhel,multi_platform_fedora
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name: "Set SSH Maximum Idle Keepalive Count"
+  lineinfile:
+    create=yes
+    dest="/etc/ssh/sshd_config"
+    regexp="^ClientAliveIntervalMax\s"
+    line="ClientAliveIntervalMax (ansible-populate sshd_set_keepalive)"
```

Comment:
Is the XCCDF Value for `(ansible-populate sshd_set_keepalive)` declared anywhere? I can't find it in `shared/xccdf`.